File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/bass/ComBizLoadIFS0001Info.java
package nta.med.data.model.ihis.bass;

public class ComBizLoadIFS0001Info {
	private String codeType;
	private String codeTypeName;
	private String remark;
	private String sysDate;
	private String sysId;
	private String updDate;
	private String updId;

	public ComBizLoadIFS0001Info(String codeType, String codeTypeName,
			String remark, String sysDate, String sysId, String updDate,
			String updId) {
		super();
		this.codeType = codeType;
		this.codeTypeName = codeTypeName;
		this.remark = remark;
		this.sysDate = sysDate;
		this.sysId = sysId;
		this.updDate = updDate;
		this.updId = updId;
	}

	public String getCodeType() {
		return codeType;
	}

	public void setCodeType(String codeType) {
		this.codeType = codeType;
	}

	public String getCodeTypeName() {
		return codeTypeName;
	}

	public void setCodeTypeName(String codeTypeName) {
		this.codeTypeName = codeTypeName;
	}

	public String getRemark() {
		return remark;
	}

	public void setRemark(String remark) {
		this.remark = remark;
	}

	public String getSysDate() {
		return sysDate;
	}

	public void setSysDate(String sysDate) {
		this.sysDate = sysDate;
	}

	public String getSysId() {
		return sysId;
	}

	public void setSysId(String sysId) {
		this.sysId = sysId;
	}

	public String getUpdDate() {
		return updDate;
	}

	public void setUpdDate(String updDate) {
		this.updDate = updDate;
	}

	public String getUpdId() {
		return updId;
	}

	public void setUpdId(String updId) {
		this.updId = updId;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Bass/ComBizLoadIFS0001Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Bass
{
    [Serializable]
    public class ComBizLoadIFS0001Info
    {
        private String _codeType;
        private String _codeTypeName;
        private String _remark;
        private String _sysDate;
        private String _sysId;
        private String _updDate;
        private String _updId;

        public String CodeType
        {
            get { return this._codeType; }
            set { this._codeType = value; }
        }

        public String CodeTypeName
        {
            get { return this._codeTypeName; }
            set { this._codeTypeName = value; }
        }

        public String Remark
        {
            get { return this._remark; }
            set { this._remark = value; }
        }

        public String SysDate
        {
            get { return this._sysDate; }
            set { this._sysDate = value; }
        }

        public String SysId
        {
            get { return this._sysId; }
            set { this._sysId = value; }
        }

        public String UpdDate
        {
            get { return this._updDate; }
            set { this._updDate = value; }
        }

        public String UpdId
        {
            get { return this._updId; }
            set { this._updId = value; }
        }

        public ComBizLoadIFS0001Info() { }

        public ComBizLoadIFS0001Info(String codeType, String codeTypeName, String remark, String sysDate, String sysId, String updDate, String updId)
        {
            this._codeType = codeType;
            this._codeTypeName = codeTypeName;
            this._remark = remark;
            this._sysDate = sysDate;
            this._sysId = sysId;
            this._updDate = updDate;
            this._updId = updId;
        }

    }
}